src/mono/wasm/wasm.targets

```diff
@@ -32,9 +34,20 @@
                            OutputPath="$(WasmPInvokeTablePath)" />
   </Target>
 
+  <Target Name="BundleWasm" DependsOnTargets="CheckEnv">
```

Comment:
Nit: this target is bundling the timezone, should we rename the target, or are we expecting to add more calls to this task with a different bundle `type`?

Reply:
I believe we eventually want this task to be used with other data types besides just the TimeZone data.